core/src/main/java/cz/xtf/core/openshift/OpenShiftBinary.java

```diff
@@ -52,7 +53,15 @@ public String execute(String... args) {
         if (ocConfigPath == null) {
             return executeCommand(ArrayUtils.addAll(new String[] { path }, args));
         } else {
-            return executeCommand(ArrayUtils.addAll(new String[] { path, "--config=" + ocConfigPath }, args));
+            log.debug("openshift version: [{}]", OpenShifts.getVersion());
```

Comment:
Problem with `OpenShifts.getVersion()` is that it returns version of OCP cluster and not the `oc` client. There are two options, add logic to get oc client version or just use `--kubeconfig=` for all `oc` versions. I would prefer 2nd option as we don't need to mess up the code by IFs for given versions. wdyt?

Reply:
by default, XTF downloads client version corresponding to cluster version, but yeah, it is issue when version is overriden in config.

Reply:
I like the second option and it is compatible with 3.11 too. Going for update PR

Reply:
thanks